README.md

```diff
@@ -1 +1,2 @@
-# pilz_common
\ No newline at end of file
+# pilz_common
```

Comment:
I suppose we should bring this a little more in line with other READMEs we have in our repos.
I will commit a suggestion soon.